pipelines/resources/edgeclusters/edgecluster-deploy-lso.yaml

```diff
@@ -48,8 +36,12 @@ spec:
         #!/usr/bin/bash
 
         if [[ "${MOCK}" == 'false' ]]; then
-          cd ${WORKDIR}/${DEPLOY_ODF_DIR}
-          ./deploy.sh
+          ztp create lso \
+          --config "$(params.edgeclusters-config)" \
+          --wipe \
```

Comment:
https://issues.redhat.com/browse/MGMT-13974

Reply:
Done, removed that option from the CLI and here.